src/buffer/out/AttrRow.cpp

```diff
@@ -192,6 +192,54 @@ std::unordered_set<uint16_t> ATTR_ROW::GetHyperlinks()
     return ids;
 }
 
+// Method description:
+// - Gets the start and endpoint of a particular link in this row, ignoring other attributes
+// - Use this to get the full hyperlinked text from just one cell (for example, if the user hovers over a
+//   cell in the middle of a link and we need the entire link)
+// Arguments:
+// - The column in the link
+// Return value:
+// - A pair where the first value is the start point and the second value is the endpoint of the link

```

Comment:
**NOTE**

Currently, this function is not called anywhere. I wrote it because I thought it might be useful to have the border cover the entire link instead of the cell that the user is hovering over, but we do not actually need that for now. 

Regardless, I thought this function could potentially be useful in the future so I left it in. Can remove it if anyone feels strongly about it!

Reply:
Slap a unit test on it and I won't cry if you leave it in.

Reply:
This one may not be exactly what we need -- the same link can be present on multiple rows, and a function existing that returns just one row might lull someone into a false sense of security/a sense of "having found the right way to do it" later on

Reply:
Okay, I'll just remove it!